src/rules/prefer-jest-globals.ts

```diff
@@ -0,0 +1,60 @@
+import globalsJson from '../globals.json';
+import { createRule } from './utils';
+
+export default createRule({
+  name: __filename,
+  meta: {
+    docs: {
+      category: 'Best Practices',
+      description: 'Prefer importing Jest globals',
+      recommended: false,
+    },
+    messages: {
+      preferJestGlobal:
+        "Jest function \"{{ jestFunction }} is used but not imported from '@jest/globals'",
+    },
+    fixable: 'code',
+    type: 'suggestion',
+    schema: [],
+  },
+  defaultOptions: [],
+  create(context) {
+    const jestFunctions = Object.keys(globalsJson);
+    const importedJestFunctions: any[] = [];
+    const usedJestFunctions = new Set();
+
+    return {
```

Comment:
Can you look at using `parseJestFnCall` for this? it already handles determining if a Jest function is imported or not as part of its core logic, so it should greatly simplify the logic

Reply:
yep, refactored using `parseJestFnCall`